flake.nix

```diff
@@ -1,87 +1,40 @@
 {
   description = "Steel";
 
-  inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable"; # We want to use packages from the binary cache
-    flake-utils.url = "github:numtide/flake-utils";
-  };
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
 
   outputs = {
     self,
     nixpkgs,
-    flake-utils,
-  }:
-    flake-utils.lib.eachDefaultSystem (system: let
-      pkgs = nixpkgs.legacyPackages.${system};
-      fs = pkgs.lib.fileset;
+  }: let
+    systems = ["x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin"];
```

Comment:
Add `inputs.systems.url = "github:nix-systems/default";` and use `systems = import systems;` instead.

This way users can override systems to try and build on other platforms.